<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferralEditDialog</class>
 <widget class="QDialog" name="ReferralEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Направление</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="lblSrcNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Номер</string>
     </property>
     <property name="buddy">
      <cstring>edtSrcNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSrcOrg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Направитель</string>
     </property>
     <property name="buddy">
      <cstring>cmbSrcOrg</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="CDateEdit" name="edtSrcDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblSrcDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Дата</string>
     </property>
     <property name="buddy">
      <cstring>edtSrcDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="CPolyclinicComboBox" name="cmbSrcOrg"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnSelectSrcOrg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="edtSrcNumber">
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSrcSpeciality">
     <property name="text">
      <string>&amp;Специальность</string>
     </property>
     <property name="buddy">
      <cstring>cmbSrcSpeciality</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbSrcSpeciality"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSrcPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>edtSrcPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="edtSrcPerson">
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPolyclinicComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbSrcOrg</tabstop>
  <tabstop>btnSelectSrcOrg</tabstop>
  <tabstop>edtSrcPerson</tabstop>
  <tabstop>cmbSrcSpeciality</tabstop>
  <tabstop>edtSrcNumber</tabstop>
  <tabstop>edtSrcDate</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReferralEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReferralEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
